sql/moz-fx-data-shared-prod/telemetry_derived/investigation_aggregates_v1/query.sql

```diff
@@ -0,0 +1,256 @@
+    THEN
+      'Other'
+    ELSE
+      ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)]
+    END
+    AS adjust_network,
+    CASE
+    WHEN
+      ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] NOT IN (
+        'com.android.vending'
+      )
+      AND ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] IS NOT NULL
+    THEN
+      'Other'
+    ELSE
+      ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)]
+    END
+    AS install_source
+  FROM
+    org_mozilla_fennec_aurora.metrics AS org_mozilla_fennec_aurora_metrics
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+  GROUP BY
+    1
+  UNION ALL
+  -- Fenix beta
+  SELECT
+    client_info.client_id,
+    MIN(DATE(submission_timestamp)) AS submission_date,
+    CASE
+    WHEN
+      ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)] NOT IN (
+        '',
+        'Organic',
+        'Google Organic Search',
+        'Untrusted Devices',
+        'Product Marketing (Owned media)',
+        'Google Ads ACI'
+      )
+      AND ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)] IS NOT NULL
+    THEN
+      'Other'
+    ELSE
+      ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)]
+    END
+    AS adjust_network,
+    CASE
+    WHEN
+      ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] NOT IN (
+        'com.android.vending'
+      )
+      AND ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] IS NOT NULL
+    THEN
+      'Other'
+    ELSE
+      ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)]
+    END
+    AS install_source
+  FROM
+    org_mozilla_firefox_beta.metrics AS org_mozilla_firefox_beta_metrics
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+  GROUP BY
+    1
+  UNION ALL
+  -- Fenix release
+  SELECT
+    client_info.client_id,
+    MIN(DATE(submission_timestamp)) AS submission_date,
+    CASE
+    WHEN
+      ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)] NOT IN (
+        '',
+        'Organic',
+        'Google Organic Search',
+        'Untrusted Devices',
+        'Product Marketing (Owned media)',
+        'Google Ads ACI'
+      )
+      AND ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)] IS NOT NULL
+    THEN
+      'Other'
+    ELSE
+      ARRAY_AGG(metrics.string.metrics_adjust_network)[SAFE_OFFSET(0)]
+    END
+    AS adjust_network,
+    CASE
+    WHEN
+      ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] NOT IN (
+        'com.android.vending'
+      )
+      AND ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)] IS NOT NULL
+    THEN
+      'Other'
+    ELSE
+      ARRAY_AGG(metrics.string.metrics_install_source)[SAFE_OFFSET(0)]
+    END
+    AS install_source
+  FROM
+    org_mozilla_firefox.metrics AS org_mozilla_firefox_metrics
+  WHERE
+    DATE(submission_timestamp) >= '2021-01-01'
+  GROUP BY
+    1
+),
+fenix AS (
```

Comment:
Is this data specific only to fenix? If that is the case perhaps we should rename the table to reflect that?

Reply:
It only contains data for Fenix now, but my understanding is that it is in the board to collect attribution for other browsers.
So my plan is to keep this as a unique aggregate for investigations and add data as it is made available, instead of creating separate datasets or change the models build on top of it if we rename the table later.
Another option is to add the browser_name column already, which would be an unnecessary storage cost but mitigates future changes.

What is your preference?

Reply:
Investigations as in debugging? I wonder if maybe in that case it should live in a different dataset? I think we should avoiding production tables and debug tables together.